packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/WorkflowSelector.js

```diff
@@ -76,11 +80,16 @@ function WorkflowSelector (props) {
 }
 
 WorkflowSelector.propTypes = {
+  workflowDescription: string,
   workflows: shape({
     data: arrayOf(shape({
       id: string.isRequired
     }).isRequired).isRequired
   }).isRequired
 }
 
+WorkflowSelector.defaultProps = {
+  workflowDescription: counterpart('WorkflowSelector.message')
```

Comment:
Can this be overridden by supplying an empty string as the `workflowDescription` prop?